### PR TITLE
en-GB style guide says ID not id

### PR DESCRIPTION
**Pull request by brianteeman**
The style guide says to use ID https://developer.joomla.org/en-gb-user-interface-text-guidelines/words-to-watch.html

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_weblinks.ini [hunk 1]
```diff
@@ -81,7 +81,7 @@ COM_WEBLINKS_FIELD_WIDTH_LABEL="Width"
 COM_WEBLINKS_FIELDSET_IMAGES="Images"
 COM_WEBLINKS_FIELDSET_OPTIONS="Options"
 COM_WEBLINKS_FILTER_CATEGORY="Filter Category"
-COM_WEBLINKS_FILTER_SEARCH_DESC="Search in web link title and alias. Prefix with id: to search for an web link ID."
```
Comment (yvesh) [inline administrator/language/en-GB/en-GB.com_weblinks.ini]: Can you also fix this directly in the weblinks language files? :-)

**Review by yvesh — APPROVED** [review]

File: administrator/language/en-GB/en-GB.com_weblinks.ini (hunk 1 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_weblinks.ini]: I think you will find it is the other way around. @zero-24 fixed it in the web links repo but didnt fix it here https://github.com/joomla-extensions/weblinks/pull/315

Comment (yvesh) [inline administrator/language/en-GB/en-GB.com_weblinks.ini]: Oh, you are right. Thought he wanted to open a pull request.. But maybe he forgot. :-) Thanks for fixing!

Comment (zero-24) [inline administrator/language/en-GB/en-GB.com_weblinks.ini]: Maybe i need more sleep but i have moved any change from joomla-extensions in this repo here.

If you check the PR you linked you notice that i have changed it from `ID` to `id` there or I#m just blind?

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_weblinks.ini]: yes you changed it to ID in the wbelinks repo but you didnt make the change here - thats what this PR does

Comment (wilsonge) [inline administrator/language/en-GB/en-GB.com_weblinks.ini]: In the PR you linked to Brian it's been moved to lowercase

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_weblinks.ini]: Ah I see - that was wrong - I will change it back

**Pull request merged**